develop/components/pages/DropdownPage.tsx

```diff
@@ -11,6 +11,11 @@ const DropdownPage: React.FunctionComponent = () => {
     const [dropDownList4Selected, setDropdownList4Selected] = React.useState<DropdownItem>(null);
     const [dropDownList5Selected, setDropdownList5Selected] = React.useState<Array<DropdownItem>>([]);
     const [disabled, setDisabled] = React.useState<boolean>(false);
+    const [error, setError] = React.useState<string>("");
+
+    const handleToggleError = React.useCallback(() => {
+        setError((currentError) => (currentError === null) ? "Example error message" : null);
```

Comment:
What is this `currentError` and where is it coming from ?

Reply:
The React useState setter (setError in thius case) accepts a new value **or alternatively** a function which provides the current state and lets you return a new state.

Reply:
https://reactjs.org/docs/hooks-reference.html#functional-updates